### PR TITLE
windows-compatible build script

### PR DESCRIPTION
**Pull request by zeowang**
Added powershell script enables build on windows. (fixing the problem that blob syntax** is not usable in windows powershell)
Added new npm script option for building use the powershell script.

Run `npm run build-win` to use the script.

Note: `npm run build` will build the file `/src/foo/bar.tx` into `/dist/foo/bar/index.js`. Where as this script will build `/src/foo/bar.tx` into `/dist/foo/bar.js` which might be a little bit different.

### CODE REVIEW

File: scripts/serve [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+echo "Starting local development environment..."
+# compile TS to JS with hot reload
+esbuild ./src/**/*.ts \
+  --entry-names=[dir]/[name]/index \
+  --bundle \
+  --platform=node \
+  --target=node16.14 \
+  --outdir=./dist/ \
+  --watch=forever &
+
```
Comment (armintalaie) [inline scripts/serve:31]:
```suggestion
  esbuild_pid=$!
  trap "kill $esbuild_pid" SIGINT
```

File: package.json [hunk 2]
```diff
@@ -7,7 +7,8 @@
     },
     "scripts": {
         "clean": "rm -rf dist",
-        "build": " esbuild ./src/**/*.ts --entry-names=[dir]/[name]/index --bundle   --platform=node --target=node16.14 --outdir=./dist/ ",
+        "build": " sh ./scripts/build", 
+        "TEST:API": " sh ./scripts/serve",
```
Comment (armintalaie) [inline package.json:11]:
```suggestion
        "test:api": " sh ./scripts/serve", 
```

File: scripts/serve (hunk 1 above)
Comment (armintalaie) [reply scripts/serve:31]: ctrl^c would not kill the port. i think this oughta do it

**Review by armintalaie — CHANGES_REQUESTED** [review]
It looks good to me. This should be good to make it a bit simpler to run, fix the windows issue, also adds hot reload

please update the 
> - To run your API locally with a debugger, run `sam local start-api -d 5858`
in the `docs/developer.md` with better instructions to run the API.

**Comment by armintalaie** [comment]
@zeowang any updates on this?

**Comment by zeowang** [comment]
Had added the passing parameter: when invoking npm run test:api, one can add param like `-- wiki.stack-template 8000 LAZY` to indicate parameters. If nothing is added then it will use some default template-port-dockerOption parameters in the script.

Hot reload works, just some minor issue left on ctrl-c to end the background server...

**Review by armintalaie — APPROVED** [review]

**Pull request merged**